src/sstruct_ls/sys_pfmg_solve.c

```diff
@@ -47,14 +47,19 @@ hypre_SysPFMGSolve( void                 *sys_pfmg_vdata,
    HYPRE_Real             *rel_norms       = (sys_pfmg_data -> rel_norms);
    HYPRE_Int              *active_l        = (sys_pfmg_data -> active_l);
    HYPRE_Int               logging         = (sys_pfmg_data -> logging);
-   HYPRE_Int               print_level     = (sys_pfmg_data -> print_level);
-   HYPRE_Int               print_freq      = (sys_pfmg_data -> print_freq);
 
-   HYPRE_Real            b_dot_b, r_dot_r, eps = 0;
-   HYPRE_Real            e_dot_e = 0, x_dot_x = 1;
+   HYPRE_Real              b_dot_b, r_dot_r, eps = 0;
+   HYPRE_Real              e_dot_e = 0, x_dot_x = 1;
 
-   HYPRE_Int             i, l;
-   char                  filename[255];
+   HYPRE_Int               i, l;
+#if 0
+   HYPRE_Int               print_level     = (sys_pfmg_data -> print_level);
```

Comment:
It creates a compile warning because line 338 is turned off now.  The reason for the latter is to match the behavior of PFMG (it also eliminates a residual computation, which we can debate the pros and cons of), which is one of the regression tests.

Reply:
I see now, thanks Rob!